src/main/resources/liquibase/202310131200-remove-account-config.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+  <changeSet id="202310131000-01" author="vbusch">
+    <comment>Drop account_config table.  org_conf is used instead.</comment>
+    <dropTable tableName="account_config"/>
+    <rollback>
+      <createTable tableName="account_config">
+        <column name="account_number" type="VARCHAR(255)">
+                <constraints nullable="true" unique="true" uniqueConstraintName="account_config_account_number_unq"/>
+        </column>
+        <column name="opt_in_type" type="VARCHAR(255)">
+          <constraints nullable="false" />
+        </column>
+        <column name="created" type="TIMESTAMP WITH TIME ZONE">
+          <constraints nullable="false" />
+        </column>
+        <column name="updated" type="TIMESTAMP WITH TIME ZONE">
+          <constraints nullable="false" />
+        </column>
+        <column name="org_id" type="varchar(32)">
+          <constraints primaryKey="true" primaryKeyName="account_config_pkey" />
+        </column>
+      </createTable>
+
+    </rollback>
+  </changeSet>
+
+  <changeSet id="202310031000-02" author="vbusch" dbms="postgresql">
+    <comment>Drop account_number column from subscription table</comment>
+    <dropColumn tableName="subscription" columnName="account_number"/>
+
+    <rollback>
+      <addColumn tableName="subscription"> <column name="account_number" type="varchar(255)"/> </addColumn>
+
+      <createIndex tableName="subscription" indexName="subscription_account_number_sku_idx">
+        <column name="account_number"/>
+        <column name="sku"/>
+      </createIndex>
+    </rollback>
+  </changeSet>
+
+  <changeSet id="202310031000-03" author="vbusch" dbms="postgresql">
+    <comment>Drop account_number column from hosts table</comment>
+    <dropColumn tableName="hosts" columnName="account_number"/>
+
+    <rollback>
+      <addColumn tableName="hosts"> <column name="account_number" type="varchar(255)"/> </addColumn>
+
+      <createIndex indexName="account_idx" tableName="hosts">
+        <column name="account_number"/>
+      </createIndex>
+    </rollback>
+  </changeSet>
+
+  <changeSet id="202310031000-04" author="vbusch" dbms="postgresql">
```

Comment:
`dbms="postgresql"` is necessary here because there is a constraint that was created without a name: https://github.com/RedHatInsights/rhsm-subscriptions/blob/bcfa5919f7bc1c64b469d27ba19e5048601c3e65/src/main/resources/liquibase/202102251446-add-constraints-indexes-fields-to-events.xml#L22

In PostgreSQL, the constraint name are generated using the column names. In HSQL, it's using a SYS_XX name convention. We should be able to configure it using `sql.sys_index_names` (see http://hsqldb.org/doc/guide/dbproperties-chapt.html#dpc_sql_conformance).